src/runtime/rpc/network.cpp

```diff
@@ -590,6 +590,11 @@ connection_oriented_network::connection_oriented_network(rpc_engine *srv, networ
     : network(srv, inner_provider)
 {
     _cfg_conn_threshold_per_ip = 0;
+    _client_session_count.init_global_counter("replica",
+                                              "network",
+                                              "server_session_count",
+                                              COUNTER_TYPE_NUMBER,
+                                              "current session count on server");
```

Comment:
I think `server.network.client_session_count` maybe better. 
Firstly, both replica server and meta server will share this counter, `server` maybe more suitable.  Besides, `client_session_count` or `session_count` is better.